app/javascript/channels/instance_channel.js

```diff
@@ -0,0 +1,26 @@
+import consumer from "./consumer";
+
+const initInstanceChannel = () => {
+  console.log("Connection complete");
+  const instanceContainer = document.getElementById("instance")
+  if (instanceContainer) {
+    const id = instanceContainer.dataset.instanceId;
+    console.log("Instance id: ", id);
+
+    consumer.subscriptions.create({ channel: "InstanceChannel" }, {
```

Comment:
We seem to be missing to link the channel id here from line 7. [Video reference from lecture](https://kitt.lewagon.com/camps/718/lectures/06-Projects%2F01-Pundit#video) at time: 40:40.

Not having the id is ok if we always expect there to always only be one channel in the app. However multiple channels/dbi may be created at a time.

Reply:
Ah! You're right! I forgot to add that back in. Thanks for the catch.